tests/tensorflow/README.md

```diff
@@ -0,0 +1,26 @@
+
+```
+pytest tests/common tests/tensorflow \
+         --junitxml nncf-tests.xml \
+	 --data tests/tensorflow/data
+```
+
+The test result will be saved in `nncf-test.xml`, and it will returns pass, skip, warnings or error during the test. Also, here are some explanation about the test files.
+
+### test_sanity_sample.py
+In this file, you will test the basic training and evalutation loop in NNCF. The `generate_config_params` function will generate some test configs that will be tested, and it will be saved into `CONFIG_PARAMS` in `list` shape, and each element is `tuple` type. One example in `CONFIG_PARAMS` is like:
+`('classification', '{nncf-dir}/tests/tensorflow/data/configs/resnet50_cifar10_magnitude_sparsity_int8.json', 'cifar10', 'tfrecord')`.
+When the list of sample configuration generated, it will saved into `_config` with dictionary type.
+
+The functions `test_model_eval`, `test_model_train`, `test_trained_model_eval`, or other similar functions are the key functions in this file. It receives parameters from config which is generated as sample, and temporary path that will be generated and deleted only for test. The variable `main` in this function will get main function which is defined in each task(e.g. for classification: `examples/tensorflow/classification/main.py`). And dict type `args` will make arguments needed for main function based on the configs, so that it changed to argv and it will be used to run the `main`. Each function will test the model from checkpoint, or 1~2 epochs of training, or export to onnx.
+
+### test_sota_checkpoints.py
+In this file, you can test whether the models with checkpoints match the published sota performance. You can see the sota configurations are written in `sota_checkpoints_eval.json`, which contains the tasks / datasets / topologies. In topologies, it contains model name as a key and various datas such as config file path, ckpt path, target performance based on metric_type, compression method or etc. The datas from the json file will added to the class `EvalRunParamsStruct`, and the configs will be ran in `test_openvino_eval`.
+
+### test_weekly.py
+
+In this file, you will test the tasks or models in `GLOBAL_CONFIG`. The `tolerance` term is the term on how much error to allow for relative accuracy, with the default value of 0.5. For example, if the expected accuracy is 75 and the tolerance value is 0.5, then an accuracy between 74.5 and 75.5 is allowed for test. The `run_sample` function get the `main` function defined in the config and run the test. It will compare the `actual_acc` and `ref_acc` so that the absolute difference the to value are in `tolerence` value. When the `--run-weekly-tests` parameter is entered, it consequently run the train and test mode from `run_sample` function.
```

Comment:
Would you please modify these paragraphs to make it clearer based on the following requirements?

1. Explain why this test is needed in the first sentence
2. Highlight the differences among them